.scrutinizer.yml

```diff
@@ -1,9 +1,30 @@
-tools:
-    external_code_coverage: true
+build:
```

Comment:
do not like the idea of running tests both on travis and scrutinizer... is necessary to configure two envs...
sending code coverage from travis to scrutinizer is a common practice. 
any reason for it? am I missing something?

Reply:
solely for avoiding setting up & waiting for travis to complete a run before getting feedback from scrutinizer; this can be rolled back if you'd prefer :)

Reply:
> solely for avoiding setting up & waiting for travis to complete a run before getting feedback from scrutinizer; this can be rolled back if you'd prefer :)

yes, thanks :)

Reply:
handled in 3143682